frappe/public/js/frappe/file_uploader/FileUploader.vue

```diff
@@ -329,9 +338,17 @@ export default {
 
 			if (!is_correct_type) {
 				console.warn('File skipped because of invalid file type', file);
+				frappe.show_alert({
+					message:__(`File "${file.name}" was skipped because of invalid file type`),
```

Comment:
```suggestion
					message: __('File "{0}" was skipped because of invalid file type', [file.name]),
```

template strings are not valid in translation syntax

https://frappeframework.com/docs/user/en/translations#2-variables